src/main/java/com/global/api/gateways/RealexConnector.java

```diff
@@ -303,10 +303,14 @@ else if(builder.getPaymentMethod() instanceof RecurringPaymentMethod) {
             if(builder.getShippingAddress() != null)
                 tssInfo.append(buildAddress(et, builder.getShippingAddress()));
         }
+        
+        if (builder.getEstimatedNumTxn() != null) {
+                et.subElement(request, "estnumtxn").text(Integer.toString(builder.getEstimatedNumTxn()));
```

Comment:
The ElementTree object already handles nulls so you can simply call:
et.subElement(request, "estnumtxn", builder.getEstimatedNumberOfTransactions();

or in the case of the single "multi capture options":
if(builder.getMutliCaptureOptions() != null) { /* add all elements here */ }